tests/bootstrap.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+require_once __DIR__.'/../vendor/autoload.php';
+
+use SebastianBergmann\Comparator\Factory;
+use tests\Libero\ContentApiBundle\ResourceComparator;
+
+Factory::getInstance()->register(new ResourceComparator());
```

Comment:
As mentioned in https://github.com/libero/content-api-bundle/pull/5#discussion_r229208333.